operators/volume_renderer/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ set(CLARA_VIZ_WITH_EXAMPLES OFF CACHE INTERNAL "")
 # we are downloading large content, it's better to see the progress
 set(FETCHCONTENT_QUIET OFF)
 
+# enable CMP0135 policy to use timestamp when fetching content
+cmake_policy(SET CMP0135 NEW)
```

Comment:
I'm also seeing this `CMP0135` warning with the volume_loader op when fetching `nifti_clib`. Probably needs to be added there as well.

Reply:
I don't see it, but added as well.